quark/subproject.py

```diff
@@ -5,6 +5,7 @@
 from shutil import rmtree
 from subprocess import call, PIPE, Popen, CalledProcessError, run
 from urllib.parse import urlparse
+from utils import cmake_escape
```

Comment:
Just tested, apparently this must be imported as `quark.utils`, otherwise invoking quark through `bin/quark` breaks.